File: src/main/java/com/everestengineering/employeeportalapplication/model/Employee.java
package com.everestengineering.employeeportalapplication.model;

import lombok.Data;

import java.util.Date;

@Data
public class Employee {
    private Long empID;
    private String firstName;
    private String lastName;
    private String everestEmailId;
    private String password;
    private String personalEmailId;
    private Date dateOfBirth;
    private Date dateOfJoin;
    private String designation;
    private String priorExperienceInYears;
    private String bio;
    private EmployeeAddress employeeAddress;
    private EmployeeAddress permanentAddress;

}

File: src/frontend/src/Employee.js
import React from "react";

const Employee =(props)  =>{
  <div>
    <p>
      <b>Name:</b>{props.firstName} {props.lastName}
    </p>
    <p><b>Email:</b>{props.everestEmailId}</p>
  </div>

}

export default Employee;